app/src/main/java/org/mozilla/focus/fragment/ZxingActivity.java

```diff
@@ -0,0 +1,70 @@
+package org.mozilla.focus.fragment;
+
+import android.Manifest;
+import android.content.Intent;
+import android.content.pm.PackageManager;
+import android.os.Bundle;
+import android.support.v4.app.ActivityCompat;
+import android.support.v4.content.ContextCompat;
+import android.support.v7.app.AppCompatActivity;
+import android.support.v7.widget.Toolbar;
+import android.view.View;
+import android.widget.Toast;
+
+import org.mozilla.focus.R;
+
+/**
+ * Created by Achintya on 9/26/2017.
+ */
+
+public class ZxingActivity extends AppCompatActivity {
```

Comment:
\[PMD\] INFO: The class 'ZxingActivity' has a Cyclomatic Complexity of 2 \(Highest = 5\)\.